parsel/selector.py

```diff
@@ -90,9 +90,15 @@ def create_root_node(
     parser_cls: Type[_ParserType],
     base_url: Optional[str] = None,
     huge_tree: bool = LXML_SUPPORTS_HUGE_TREE,
+    body: Optional[bytes] = "",
```

Comment:
If there is no use for body being `None`, we may as well not allow it in the first place. And the default value should match the type hint.

```suggestion
    body: bytes = b"",
```

Reply:
Fixed